src/web-ui/README.md

```diff
@@ -22,3 +22,46 @@ foo@bar:~$ docker-compose up --build web-ui
 ```
 
 Once the container is up and running, you can access it in your browser at [http://localhost:8080](http://localhost:8080).
+
+# Site Optimization with Moovweb XDN
+
+Read more information about XDN [here](https://developer.moovweb.com/guides/starter)
+
+## Links
+
+Project: https://moovweb.app/moovweb-demos/aws-rds
```

Comment:
Should developers/users of the Retail Demo Store be able to sign in to this account? Should they have their own XDN account?

Reply:
Thanks for noticing that, I missed these links in readme which I just had in my private project. Removed all unnecessary links.